Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -72,7 +72,10 @@ extension SwiftDoc {
           pages[path(for: name)] = GlobalPage(module: module, name: name, symbols: symbols)
         }
 
-        guard !pages.isEmpty else { return }
+        guard !pages.isEmpty else {
+            logger.warning("No pages were found to render.")
```

Comment:
```suggestion
            logger.warning("No public API symbols were found at the specified path. No output was written.")
```

Reply:
Language to be updated when we support variable ACL (#91). Feel free to pitch any alternatives you can think of. I just wanted to be a bit more explicit about cause and effect in the error message.

Reply:
WFM!